pages/common/traceroute.md

```diff
@@ -18,6 +18,6 @@
 
 `traceroute -q 5 {{host}}`
 
-- Specify size in bytes of datagram:
+- Specify size in bytes of probing packet:
 
-`traceroute {{host}} {{size<=32768}}`
+`traceroute {{host}} 42`
```

Comment:
> fixing it on this PR, regardless of who does it.

Actually, only @zkendall can edit this PR's branch, because it doesn't exist in the tldr-pages repo, but on his fork (where only he has commit rights).

If we are to fix it ourselves, it would be easier to merge the PR and then edit the page, than manually create a branch and open a new PR based on the changes of this one (which would mean this PR would have to be closed as the contents would be duplicated in the new branch)... it isn't worth the hassle IMO. So either we merge this and make the fix ourselves directly on master, or wait for @zkendall to say whether he's up for it.